# HG changeset patch
# User Mark Lodato <lodatom@gmail.com>
# Date 1254710088 14400
freeze: rename README.rst to README.txt

The only reason I had a "rst" extension was so that GitHub would pick it
up.

diff --git a/Demos/freeze/README.rst b/Demos/freeze/README.txt
similarity index 100%
rename from Demos/freeze/README.rst
rename to Demos/freeze/README.txt
diff --git a/bin/cython_freeze b/bin/cython_freeze
index 6e47586..0709ee5 100755
--- a/bin/cython_freeze
+++ b/bin/cython_freeze
@@ -3,7 +3,7 @@
 Create a C file for embedding one or more Cython source files.
 Requires Cython 0.11.2 (or perhaps newer).
 
-See Demos/freeze/README.rst for more details.
+See Demos/freeze/README.txt for more details.
 """
 
 import optparse
-- 
1.6.5.rc3

